vnext/Microsoft.ReactNative.Managed.CodeGen/OlderNetCoreVersionHacks.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
>// [](start = 0, length = 2)

Please do not use word 'hack'. It is on the 'do not use list' in the Polycheck tool. Consider using 'Workaround' or something else along these lines.